src/components/forms/pagination/pagination-form.view.js

```diff
@@ -0,0 +1,120 @@
+const PaginationForm = Backbone.View.extend(
+    /** @lends PaginationForm.prototype */
+    {
+        /**
+         * This will append the tagName and className
+         * along with the current one into the DOM
+         * @member {String} tagName - the tag of the element
+         * @member {String} className - the class attribute of the element
+         */
+        tagName: 'div',
+        className: 'pagination',
+        /**
+         * Search event.
+         * @event SearchPage#showPrev
+         * @event SearchPage#showMore
+         * @event SearchPage#showNext
+         */
+
+        events: {
+
+            'click [data-id = "prev"]': 'showPrev',
+            'click [data-id = "more"]': 'showMore',
+            'click [data-id = "next"]': 'showNext'
+        },
+        /**
+         * Creates a new PaginationForm instance
+         * @constructs
+         * @extends Backbone.View
+         */
+        initialize: function(params) {
+            this.params = params;
+            this.params.page = Number(this.params.page);
+        },
+        /**
+         * This will append the html from file pagination-form.html
+         * along with the current one into the DOM
+         * @returns {Object} - html from pagination-form.html
+         */
+        render: function() {
+            $.get('src/components/forms/pagination/pagination-form.html').done(tpl => this.$el.html(_.template(tpl)(this)));
+            return this;
+        },
+
+        /**
+         * Method that showing the 10 previous books
+         * @fires SearchPage#showPrev
+         */
+        showPrev: function(e) {
+            if (this.params.page === 1) return;
+            this.params.page--;
+            Backbone.history.navigate(`search/${this.params.query}/${this.params.page}`);
+            this.removePanels();
+            this.checkData((this.params.page * STEP) - STEP);
+        },
+        /**
+         * Method that adds 10 following books
+         * @fires SearchPage#showMore
+         */
+        showMore: function() {
+            Backbone.history.navigate(`search/${this.params.query}/${this.params.page + 1}`);
```

Comment:
You don't need to increment page twice. Just do ++this.params.page before navigate.